app/api/websocket.py

```
@@ -0,0 +1,316 @@
+            recent_sentiments = await sentiment_service.get_recent_sentiment(limit=10)
+            await manager.send_personal_message(
+                json.dumps(
+                    {
+                        "type": "initial_data",
+                        "data": {"recent_sentiments": recent_sentiments},
+                        "timestamp": datetime.utcnow().isoformat(),
+                    }
+                ),
+                websocket,
+            )
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)